src/routes.js

```diff
@@ -3,77 +3,96 @@ import React from 'react';
+          <Redirect from="direct" to={"/" + locale + `/encrypt/direct/${encryptVideos.length}`} />
+          <Redirect from="social" to={"/" + locale + `/encrypt/social/${encryptVideos.length}`} />
+          <Redirect from="hybrid" to={"/" + locale + `/encrypt/hybrid/${encryptVideos.length}`} />
+          <Route path="2" onEnter={redirect(locale)} />
+          <Route path="3" onEnter={redirect(locale)} />
+          <Redirect from="*" to={"/" + locale + "/encrypt/"} />
+        </Route>
+      </Route>
+    );
+  });
+  return routes;
+}
+// just need to handle / redirect now
 module.exports = (
   <Route path="/">
-    <IndexRoute component={require(`./pages/home.js`)}/>
-    <Route path="open-web-fellows">
-      <IndexRoute component={require('./pages/open-web-fellows/overview.js')}/>
-      <Route path="fellows" component={require(`./pages/open-web-fellows/fellows.js`)}/>
-      <Route path="info" component={require(`./pages/open-web-fellows/info.js`)}/>
-      <Redirect from="*" to="/open-web-fellows/" />
-    </Route>
-    <Route path="stay-secure">
-      <IndexRoute component={require(`./pages/stay-secure.js`)}/>
-      <Route path="thank-you" component={require(`./pages/stay-secure-thank-you.js`)}/>
-      <Redirect from="*" to="/stay-secure/" />
-    </Route>
-    <Route path="encrypt-hard-drive">
-      <IndexRoute component={require(`./pages/encrypt-hard-drive/encrypt-hard-drive.js`)}/>
-      <Route path="thank-you" component={require(`./pages/encrypt-hard-drive/encrypt-hard-drive-thank-you.js`)}/>
-      <Redirect from="*" to="/encrypt-hard-drive/" />
-    </Route>
-    <Route path="encrypt">
-      <IndexRoute onEnter={indexDirect} />
-      <Route path="signup" component={require(`./pages/encrypt/signup.js`)}/>
-      <Route path="signupa" component={require(`./pages/encrypt/signup-a.js`)}/>
-      <Route path="signupb" component={require(`./pages/encrypt/signup-b.js`)}/>
-      <Route path="signupc" component={require(`./pages/encrypt/signup-c.js`)}/>
-      <Route path="blanka" component={require(`./pages/encrypt/signup-blank.js`)}/>
-      <Route path="signup-complete" component={require(`./pages/encrypt/signup-complete.js`)}/>
-      <Route path="hybrida" component={require(`./pages/encrypt/hybrid-2-a.js`)}/>
-      <Route path="hybridb" component={require(`./pages/encrypt/hybrid-2-b.js`)}/>
-      <Route path="hybridc" component={require(`./pages/encrypt/hybrid-2-c.js`)}/>
-      <Route path=":type/:video" component={require('./pages/encrypt/pageType.js')}/>
-      <Redirect from="direct" to={`/encrypt/direct/${encryptVideos.length}`} />
-      <Redirect from="social" to={`/encrypt/social/${encryptVideos.length}`} />
-      <Redirect from="hybrid" to={`/encrypt/hybrid/${encryptVideos.length}`} />
-      <Route path="2" onEnter={redirect} />
-      <Route path="3" onEnter={redirect} />
-      <Redirect from="*" to="/encrypt/" />
-    </Route>
+    {buildRoutes()}
```

Comment:
We now build all of these based on the locales. This way we can know any non 404's cept "/" are good and valid.